From 51cef4d20f7fe71e8f00359f4fb5e1c3a6e570f8 Mon Sep 17 00:00:00 2001
From: Vuatech <vuatech@proton.me>
Date: Fri, 3 Oct 2025 14:53:55 -0400
Subject: [PATCH] Added fing_pkg for WaylandClientPrivate

---
 src/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 925308d..0bac937 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,3 +1,5 @@
+find_package(Qt6 REQUIRED COMPONENTS WaylandClientPrivate)
+
 qt_add_executable(hyprsysteminfo
     main.cpp
     util/Utils.cpp
